tests/acceptance/30_generic_methods/permissions_user_acl_present.cf

```diff
@@ -0,0 +1,738 @@
+}
+
+body common control
+{
+      inputs => { "${acc_path.root}/default.cf.sub", "${acc_path.root}/default_ncf.cf.sub", "@{ncf_inputs.default_files}" };
+      bundlesequence  => { configuration, default("${this.promise_filename}") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
+{
+  vars:
+      "tmp"                        string => getenv("TEMP", 1024);
+
+     # REPAIRED
+     ## recursive
+      "files[1]"     string => "file1";         # args of the GM
+      "recursive[1]" string => "true";          # \\
+      "user[1]"      string => "bin";           # \\
+      "ace[1]"       string => "+rwx";          # \\
+      "create[1]"    string => "true";          # if "true", the file will be created at init
+      "initial[1]"    slist => { "user:*:rwx", "group:*:rx", "user:bin:r", "group:bin:r", "all:r" };
+      "exp_lines[1]"  slist => { "user::rwx", "group::r-x", "user:bin:rwx", "group:bin:r--", "other::r--" };
+      # -> list of regex that must match the output of getfacl on the file at the en of the test
+      "mode[1]"      string => "enforce";       # mode, "enforce" or "audit"
+      "status[1]"    string => "repaired";      # expected status, "repaired", "success" or "error"
+      "execute[1]"   string => "true";          # Tell if an execution is needed
+
+      "files[2]"     string => "file2";
+      "recursive[2]" string => "true";
+      "user[2]"      string => "bin";
+      "ace[2]"       string => "=rw";
+      "create[2]"    string => "true";
+      "initial[2]"    slist => { "user:*:rwx", "group:*:rx", "user:bin:r", "group:bin:r", "all:r" };
+      "exp_lines[2]"  slist => { "user::rwx", "group::r-x", "user:bin:rw-", "group:bin:r--", "other::r--" };
+      "mode[2]"      string => "enforce";
+      "status[2]"    string => "repaired";
+      "execute[2]"   string => "true";
+
+      "files[3]"     string => "file3";
+      "recursive[3]" string => "true";
+      "user[3]"      string => "bin";
+      "ace[3]"       string => "-rw";
+      "create[3]"    string => "true";
+      "initial[3]"    slist => { "user:*:rwx", "group:*:rx", "user:bin:r", "group:bin:r", "all:r" };
+      "exp_lines[3]"  slist => { "user::rwx", "group::r-x", "user:bin:---", "group:bin:r--", "other::r--" };
+      "mode[3]"      string => "enforce";
+      "status[3]"    string => "repaired";
+      "execute[3]"   string => "true";
+
+      "files[4]"     string => "file4/.";
+      "recursive[4]" string => "true";
+      "user[4]"      string => "bin";
+      "ace[4]"       string => "-rw";
+      "create[4]"    string => "true";
+      "initial[4]"    slist => { "user:*:rwx", "group:*:rx", "user:bin:-rwx", "group:bin:r", "all:rx" };
+      "exp_lines[4]"  slist => { "user::rwx", "group::r-x", "user:bin:---", "group:bin:r--", "other::r-x" };
+      "mode[4]"      string => "enforce";
+      "status[4]"    string => "repaired";
```

Comment:
ok, you'll have to explain that one: it's already -rwx, so if we -rw, nothing changed